le_utils/constants/languages.py

```diff
@@ -48,16 +59,78 @@ def _initialize_language_list():
 
         yield Language(**values)
 
-
-LANGUAGE_DIRECTIONS = (
-    ('ltr', _('Left to Right')),
-    ('rtl', _('Right to Left')),
-)
-
 LANGUAGELIST = list(_initialize_language_list())
 
 _LANGLOOKUP = {l.code: l for l in LANGUAGELIST}
 
-
 def getlang(code, default=None):
+    """
+    Try to lookup a Language object for `code` in internal representation defined
+    in resources/languagelookup.json.
+    Returns None if lookup by internal representation fails.
+    """
     return _LANGLOOKUP.get(code) or None
+    # should this be   return _LANGLOOKUP.get(code) or _LANGLOOKUP[default]  ???
```

Comment:
`default` kwarg doesn't do anything right now. Maybe we could remove it to cleanup the code?